data/json/monsters/nether.json

```diff
@@ -357,6 +361,7 @@
     "weight": "200 kg",
     "hp": 350,
     "speed": 280,
+    "attack_cost": 280,
```

Comment:
Speed = attack cost pretty much neuters any hit-and-run critter (even more than the flag already does, it's a net nerf in 99% of cases).